other/materials_designer/specific_examples/defect_point_vacancy_boron_nitride.ipynb

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
<p>We should explain the units for the length. Same - for coordinate</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>